<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controls</class>
 <widget class="QWidget" name="controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame
{
	background-color: rgba(0, 0, 0, 0.6);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_play">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_play
{
	border-image: url(:/qrc/qrc/play.png);
}

#pushButton_play:unchecked
{  
	border-image: url(:/qrc/qrc/play.png);
}  
  
#pushButton_play:checked
{ 
	border-image: url(:/qrc/qrc/pause.png);
}</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_volum">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/qrc/qrc/yyf_ico_yl.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="CustomSlider" name="slider_progress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider
{
    border-color: rgba(52,7,91,100%);
}

QSlider::groove:horizontal 
{ 
	background: #cbcbcb;    
	height: 6px; 
	left: 0px; 
	right: 0px; 
	border:0px;            
	border-radius:6px; 
} 

QSlider::sub-page:horizontal 
{  
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 #0EF2D1, stop:1 #0EF2D1);  
	background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,stop: 0 #0EF2D1, stop: 1 #0EF2D1);
	border-radius: 2px;  
} 

QSlider::add-page:horizontal 
{    
	background: #cbcbcb; 
	hegiht: 4px;  
	border-radius: 2px;  
}

QSlider::handle:horizontal   
{     
	border-image: url(:/qrc/qrc/yuan.png);
    margin: -2px -4px -2px 0px; 
	width:  10px; 
	height: 10px; 
} </string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_volumNum">
        <property name="styleSheet">
         <string notr="true">#label_volumNum
{
font-size: 10px;
font-family: 微软雅黑;
color: #01EEC3;
}</string>
        </property>
        <property name="text">
         <string>00:00/00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QSlider</extends>
   <header>customslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vlcPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
